FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -110,7 +110,7 @@ extension EncryptedStorage: LogOutHandler {
             let keys = storage.objects(KeyInfo.self)
                 .filter { $0.account == email }
             let passPhrases = storage.objects(PassPhraseObject.self)
-                .filter { keys.map(\.longid).contains($0.longid) }
+                .filter { keys.map(\.primaryLongid).contains($0.longid) }
```

Comment:
Later this should be compared by `primaryFingerprint` vs `$0.fingerprint` - it's ok to do it in a followup PR